src/app/(pages)/users/[user]/components/groups/table/index.tsx

```diff
@@ -41,7 +41,7 @@ export default function GroupsTable(props: Readonly<GroupsTableProps>) {
   }
 
   const userRef: ScimReference = {
-    $ref: `${BASE_URL}/scim/Users${user.id}`,
+    $ref: `${BASE_URL}/scim/Users/${user.id}`,
```

Comment:
Probably we can recycle this function https://github.com/indigo-iam/iam-dashboard/blob/main/src/utils/scim/index.ts